bespin/collector.py

```diff
@@ -159,6 +159,7 @@ def convert_stack(path, val):
             env = configuration[["environments", environment]]
             if isinstance(env, six.string_types):
                 environment_as_dict = configuration[["environments", env]].as_dict()
+                env = configuration["environments", env]
```

Comment:
Originally thought that, but it also supports tuple.
https://github.com/delfick/bespin/pull/69/files/455a53cf201870d72f42dc1a588942c4ccc4935d#diff-721a175167557506a913d8891aa8c9d8R169